donate/views.py

```diff
@@ -11,3 +11,13 @@ class EnvVariablesView(View):
 
     def get(self, request):
         return JsonResponse(settings.FRONTEND)
+
+
+class ThunderbirdRedirectView(RedirectView):
+    """
+    A view that redirects requests to give.thunderbird.net, preserving query params
+    """
+
+    url = 'https://give.thunderbird.net/'
+    permanent = False # True
```

Comment:
got ahead of myself. Should we leave this as non-permanent so that query params are properly appended to the redirect if visited more than once on a machine?